docs/syncer.md

```diff
@@ -14,7 +14,7 @@ users have to install a special component named [syncer](https://github.com/kcp-
 1. Create a workspace and immediately enter it:
 
 ```sh
-$ kubectl kcp workspace create my-workspace --enter
+$ kubectl kcp workspace create my-workspace --type=Universal --enter
```

Comment:
is this really necessary?

Reply:
also should be lowercase, no:?

Reply:
we probably accept either case.

Reply:
afais it didn't create an universal one, let me check.

Reply:
Well if we accept both, we're still moving to a world without uppercase, so let's not use it please. Will only be confusing.

Reply:
if ommitting the `--type` it will create an org ws first, once entered it will create an universal one by default, hence i'll document this 2-step process.